### ISSUE TITLE
Question on how to dump a Cranelift module

### ISSUE THREAD

**Issue by AribYadi** [bytecodealliance/wasmtime#4707]
Hi, I need to dump/print a Cranelift module to Cranelift IR for debugging purpose, and I cannot seem to find that feature. There is `Function::display` however to dump the whole module instead of just a single function. Is there anyway to dump/print the whole module?

**Comment by bjorn3** [comment]
No, a Module contains already compiled functions. It doesn't contain clif ir. It should be possible to write a Module wrapper printing all clif ir though.
